assets/questions/plusone.js

```diff
@@ -0,0 +1,96 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Reverse Words.
+ */
+
+globalData.questions['plusone'] = {  // eslint-disable-line dot-notation
+  title: 'Increment a decimal-coded number',
+  starterCode: {
+    python:
+`def increment(s):
+    return '0'
+`
+  },
+  auxiliaryCode: {
+    python:
+`class AuxiliaryCode(object):
+    @classmethod
+    def foo(cls, array):
+        return none
+`
+  },
+  tasks: [{
+    instructions: [
+      [
+        'Given a number represented as a string, increment it.'
```

Comment:
Write a function increment that takes in a number represented as a string and increments it. This function should return the incremented number.